pkgs/native_assets_cli/example/native_add_app/pubspec_overrides.yaml

```diff
@@ -0,0 +1,3 @@
+dependency_overrides:
+  native_assets_cli:
```

Comment:
I don't see `package:native_assets_cli` depended on by the pubspec?

Reply:
I believe this was required because `native_add_library` depended on `native_assets_cli`.

Now that I've started publishing packages, I've been able to delete all the overrides. (b9ca30ca00b9d50f8e1f5ad30af98fd1602df1ab)